src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -690,11 +690,20 @@ public async Task Delete()
                     foreach (var databaseName in parameters.DatabaseNames)
                     {
                         DatabaseTopology topology = null;
-                        using (var rawRecord = ServerStore.Cluster.ReadRawDatabaseRecord(context, databaseName))
+                        var isShard = ShardHelper.TryGetShardNumberAndDatabaseName(databaseName, out string shardedDatabaseName, out int shardNumber);
```

Comment:
fine, 
but please open an issue to make a class or method out of this...
too many `ifs` for me :)